### PR TITLE
Feature/separate temps

### PR DESCRIPTION
**Pull request by tdorsey**
This adds a separate panel to distinguish between disk temperature and airflow temperature

### CODE REVIEW

**Comment by galexrt** [comment]
@tdorsey Please update / edit the existing `grafana/smartmon-textfile.json` dashboard file.

Additionally if you have time to add a screenshot of the updated dashboard to the README (the screenshot should be added to a new `media/` folder) that would be awesome.

**Comment by tdorsey** [comment]
Any particular place for the grafana version? I didn't see one listed already. Also, the dashboard format has apparently changed a ton, may want to give it a spin yourself to make sure its still loadable

**Comment by galexrt** [comment]
@tdorsey I would say in the https://github.com/galexrt/docker-node_exporter-smartmon#grafana-dashboard "Grafana Dashboard" section as an added sentence.

I will check in the next few hours.

File: README.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ The entrypoint script is putting the output into the directory `/var/lib/node_ex
 
 A Grafana dashboard for viewing the exported metrics can be found on [Grafana.com - Smartmon Textfile by Galexrt](https://grafana.com/dashboards/3992) and in the [`grafana/`](grafana/) directory.
 
+### Screenshots  
+    ![Disk and Airflow Temperatures](media/disk-airflow-temps.png?raw=true "Disk and Airflow Temps")
```
Comment (galexrt) [inline README.md]:
```suggestion
![Disk and Airflow Temperatures](media/disk-airflow-temps.png?raw=true "Disk and Airflow Temps")
```

**Comment by galexrt** [comment]
I can't check if it works because the `smartmon.sh` script doesn't seem to pickup my NVMe drives..

I have imported the dashboard and fixed the issues I saw with it (attached as a txt as JSON isn't allowed to be uploaded): [Smartmon Textfile-1603663177150.json.txt](https://github.com/galexrt/docker-node_exporter-smartmon/files/5436031/Smartmon.Textfile-1603663177150.json.txt)

I have also gone ahead and updated the `entrypoint.sh` and image to simply be a wrapper around the node_exporter textfile exporter scripts repo, so any script from that repo can be run with easy (default is still the `smartmon.sh` script to be backwards compatible).

File: README.md [hunk 2]
```diff
@@ -21,6 +21,11 @@ The entrypoint script is putting the output into the directory `/var/lib/node_ex
 
 A Grafana dashboard for viewing the exported metrics can be found on [Grafana.com - Smartmon Textfile by Galexrt](https://grafana.com/dashboards/3992) and in the [`grafana/`](grafana/) directory.
 
+Tested against Grafana version 6.1.6 and schema version 18
```
Comment (galexrt) [inline README.md:24]:
```suggestion
Tested against Grafana version `6.1.6` and schema version `18`.
```

**Comment by galexrt** [comment]
One last comment, then I'll go ahead merge the PR :slightly_smiling_face:

**Pull request closed without merging**